app/src/main/java/co/kr/mvvm_study/database/ToDoDatabase.kt

```diff
@@ -0,0 +1,25 @@
+package co.kr.mvvm_study.database
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import co.kr.mvvm_study.model.ToDoModel
+
+@Database(entities = [ToDoModel::class], version = 1)
+abstract class ToDoDatabase : RoomDatabase() {
+    abstract fun toDoDao(): ToDoDAO
+
+    companion object {
+        @Volatile
+        private var INSTANCE: ToDoDatabase? = null
+
+        fun getInstance(context: Context): ToDoDatabase = INSTANCE ?: synchronized(this) {
+            INSTANCE ?: buildDatabase(context).also { INSTANCE = it }
+        }
```

Comment:
뭔가 나라면 이부분은 아래처럼 바꾸고

```suggestion
        fun getInstance(context: Context): ToDoDatabase = INSTANCE ?: buildDatabase(context)
```